apps/full-site-editing/full-site-editing-plugin/site-editor/index.php

```diff
@@ -70,10 +70,6 @@ function enable_site_editor_experiment( $experiments_option ) {
 		$experiments_option['gutenberg-full-site-editing'] = 1;
 	}
 
-	if ( empty( $experiments_option['gutenberg-full-site-editing-demo'] ) ) {
-		$experiments_option['gutenberg-full-site-editing-demo'] = 1;
```

Comment:
is there any reason we should set this to 0? maybe old sites kept having it stick to 1?

Reply:
Maybe they are, but as I noted in Slack I'd rather have them broken and force everyone to use the new flow than having to chase bugs with various different setups.